loader/queryFromDb.go

```diff
@@ -0,0 +1,57 @@
+package loader
+
+import (
+	"database/sql"
+	"log"
+)
+
+func logError(msg string, err error) {
+	if err != nil {
+
+		log.Println("[DB]: ", msg, " Error: ", err)
+	}
+}
+
+func QueryFromDB(startTime int, endTime int, isAsc bool, dbString string) (*sql.Rows, error) {
+	db_, err := sql.Open("sqlite3", dbString)
+
+	if err != nil {
+		logError("Error connecting db ", err)
+		return nil, err
+	}
+	defer db_.Close()
+
+	order := "ASC"
+	if !isAsc {
+		order = "DESC"
+	}
+
+	stmt, err := db_.Prepare(`
+		SELECT *
+		FROM swaps
+		WHERE swap_time >= ? AND swap_time <= ?
+		ORDER BY swap_time ` + order + `
+		LIMIT 10000
```

Comment:
Should we handle more than 10k swaps here?

Reply:
For each batch size I loaded the same shard 5 times and averaged the time of ingestion there